cli/cmd/apply_cmd.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/actions"
+	"github.com/kubeshop/tracetest/cli/analytics"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var definitionFile string
+
+var applyCmd = &cobra.Command{
+	Use:    "apply [resource type]",
+	Short:  "Apply (create/update) resources to your Tracetest server",
+	Long:   "Apply resources",
+	PreRun: setupCommand(),
+	Args:   cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
In sequence for this command, we do:
1. grab the resource type
2. get the resource client
3. get the resource actions instance
4. apply the command